docs/docs/entity.md

```diff
@@ -71,14 +71,10 @@ implicit val runtime: IORuntime = cats.effect.unsafe.IORuntime.global
 ```scala mdoc:silent
 val response = Ok("").map(_.withContentType(`Content-Type`(MediaType.audio.ogg)))
 val audioDec = EntityDecoder.decodeBy(MediaType.audio.ogg) { (m: Media[IO]) =>
-  EitherT {
-    m.as[String].map(s => Audio(s).asRight[DecodeFailure])
-  }
+  m.as[String].map(s => Right(Audio(s)))
 }
 val videoDec = EntityDecoder.decodeBy(MediaType.video.ogg) { (m: Media[IO]) =>
-  EitherT {
-    m.as[String].map(s => Video(s).asRight[DecodeFailure])
-  }
+  m.as[String].map(s => Right(Video(s)))
```

Comment:
This is currently failing mdoc because of an ambiguous implicit.
You can fix this by changing the imports above:
```diff
-import cats._, cats.effect._, cats.implicits._, cats.data._
+import cats._, cats.effect._
+import cats.syntax.all._
```